service/service.go

```diff
@@ -190,9 +190,9 @@ func New(config Config) (*Service, error) {
 	var awsConfig awsclient.Config
 	{
 		awsConfig = awsclient.Config{
-			AccessKeyID:     config.Viper.GetString(config.Flag.Service.AWS.AccessKey.ID),
-			AccessKeySecret: config.Viper.GetString(config.Flag.Service.AWS.AccessKey.Secret),
-			SessionToken:    config.Viper.GetString(config.Flag.Service.AWS.AccessKey.Session),
+			AccessKeyID:     config.Viper.GetString(config.Flag.Service.AWS.HostAccessKey.ID),
+			AccessKeySecret: config.Viper.GetString(config.Flag.Service.AWS.HostAccessKey.Secret),
+			SessionToken:    config.Viper.GetString(config.Flag.Service.AWS.HostAccessKey.Session),
```

Comment:
Now we use host cluster credential, and collect default guest credential from secret (plus any other credential secret we found).